vis_services/tests/test_internals.py

```diff
@@ -300,19 +337,20 @@ def test_views_helper_functions(self):
             err = 'success'
             try:
                 res = make_request(request.request, "PN", required_fields)
-            except QueryException, err:
-                pass
-            except Exception, err:
-                pass
+            except QueryException as e:
+                err = e
+            except Exception as e:
+                err = e
             self.assertEqual(str(err), 'Cannot send both bibcodes and query')
         # No bibcodes in a bibcode query should result in an Exception
         with self.app.test_request_context(path='/wordcloud',method='POST', data=json.dumps({'bibcodes': []}), content_type='application/json') as request:
+            err = 'success'
             try:
                 res = make_request(request.request, "PN", required_fields)
-            except QueryException, err:
-                pass
-            except Exception, err:
-                pass
+            except QueryException as e:
+                err = e
+            except Exception as err:
+                err = e
```

Comment:
We capture the exception as `err` but then we use `e` within the code block.

Reply:
yes, it should have been Exception as e.